### ISSUE TITLE
Are you using public members in internal class intentionally or accidentally?!

### ISSUE THREAD

**Issue by ghalehnovy** [MarieZarrini/DataStructureExercises#2]
https://github.com/MarieZarrini/DataStructureExercises/blob/f2cab224ebde11620ca7ebb814071a74f7d77398/DataStructureExercises/SquareCalculator.cs#L3

**Read more:** https://ericlippert.com/2014/09/15/internal-or-public/

**Comment by MarieZarrini** [comment]
No, it was accidentally, that was the default protection level in Visual Studio.
